cdk/src/monitoring/synthetics.handler.ts

```diff
@@ -54,14 +60,14 @@ exports.handler = async function () {
 
   await synthetics.executeStep('viewScorecard', async function () {
     // Check for prediction presence.
-    await waitForString(page, 'Low likelihood');
+    await waitForString(page, scorecardMessages.LOW_LIKELIHOOD);
     // Check for side panel presence.
     await page.waitForSelector('.geoid-section');
     // Check for CTA presence.
-    await waitForString(page, 'Research water filters');
-    await waitForString(page, 'Copy scorecard link');
+    await waitForString(page, scorecardMessages.RESEARCH_WATER_FILTERS);
+    await waitForString(page, scorecardMessages.COPY_TO_CLIPBOARD);
     // Check for score details presence.
-    await waitForString(page, 'Understanding your score');
+    await waitForString(page, scorecardMessages.SCORECARD_SUMMARY_PANEL_HEADER(null));
```

Comment:
I think you actually do want to pass in a zip code here since it should say the zip code for Toledo

Reply:
Good point, done.

Reply:
Actually, the test fails when I pass a ZIP here. Is that expected? Here's the scorecard for that address: https://beekley.leadout-sandbox.blueconduit.com/scorecard/address/41.717025,-83.63256

(I'm going to merge this as-is just to get it in)